src/ucs/time/timerq.c

```diff
@@ -108,7 +108,7 @@ ucs_status_t ucs_timerq_remove(ucs_timer_queue_t *timerq, int timer_id)
     /* TODO realloc - shrink */
     if (timerq->num_timers == 0) {
         ucs_assert(timerq->min_interval == UCS_TIME_INFINITY);
-        free(timerq->timers);
+        ucs_free(timerq->timers);
```

Comment:
pls search "/src" for whole-world malloc/free/realloc/calloc - it should not appear there

Reply:
added more fixes in https://github.com/openucx/ucx/pull/7347